r2/r2/controllers/front.py

```diff
@@ -286,7 +286,8 @@ def GET_comments(
         infotext = None
         if request.GET.get('already_submitted'):
             submit_url = request.GET.get('submit_url') or article.url
-            resubmit_url = Link.resubmit_link(submit_url)
+            submit_title = request.GET.get('submit_title') or article.title
```

Comment:
I think it's be less confusing to not per-fill a title at all. Could you make that change?